src/CoreWCF.Primitives/src/CoreWCF/Security/ISspiNegotiation.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Runtime.CompilerServices;
+using System.Security.Authentication.ExtendedProtection;
+
+namespace CoreWCF.Security
+{
+    interface ISspiNegotiation : IDisposable
+    {
```

Comment:
This interface isn't needed as we aren't supporting the Sspi stuff yet. I have a way to do it, but it's non-trivial.